ts/sagas/profile.ts

```diff
@@ -121,7 +118,18 @@ function* createOrUpdateProfileSaga(
   const currentProfile = profileState.value;
 
   const rawAppVersion = yield* call(getAppVersion);
-  const maybeAppVersion = O.fromEither(AppVersion.decode(rawAppVersion));
+
+  const appVersion = pipe(
+    O.fromEither(AppVersion.decode(rawAppVersion)),
+    O.fold(
+      () => undefined,
+      version => version
+    )
+  );
```

Comment:
```suggestion
const appVersion = pipe(
    rawAppVersion,
    AppVersion.decode,
    E.fold(
      () => undefined,
      version => version
    )
  );
```
What do you think of this version?